packages/flutter/test/services/asset_bundle_test.dart

```diff
@@ -42,7 +40,7 @@ void main() {
 
     expect(bundle.loadCallCount['one'], 1);
 
-    Object loadException;
+    late Object loadException;
```

Comment:
Well, but these are tests.  If they are accessed without being set, it will throw, and using `late`, they are disallowed from ever being null.

Reply:
Yeah... My understanding is that at the time of `expect(loadException, isFlutterError);`, `loadException` _is_ accessed, at which time it might or might not be null. Since it's accessed when it's possibly null, it's nullable. But you're right, the result of execution is the same. Not a big deal at all.

Reply:
Agreed, not a big deal either way.